oaknational/Oak-Web-Application
# fix: 766 horizontal scroll on body on about page

## Description

Removed negative margin which was causing horizontal scroll on about page

## Issue(s)

Fixes #766            

## How to test

1. Go to {cloud link}/about-us/board
2. Make your screen 600px wide
3. You should not see any horizontal scroll

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
